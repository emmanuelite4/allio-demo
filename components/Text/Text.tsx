import styled from "@emotion/styled";

export const TextSm = styled("p")`
  font-size: 16px;
  color: var(--color-secondary);
  margin: 0;
`;

export const TextXs = styled("small")`
  font-size: 12px;
  color: var(--color-secondary);
`;
