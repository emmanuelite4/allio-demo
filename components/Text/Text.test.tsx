import { TextSm, TextXs } from "./Text";
import renderer from "react-test-renderer";

describe("Text", () => {
  it("TextSm matches snapshot", () => {
    expect(renderer.create(<TextSm />).toJSON()).toMatchSnapshot();
  });
  it("TextSm matches snapshot", () => {
    expect(renderer.create(<TextXs />).toJSON()).toMatchSnapshot();
  });
});
