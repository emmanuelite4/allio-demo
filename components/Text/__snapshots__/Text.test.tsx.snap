// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text TextSm matches snapshot 1`] = `
.emotion-0 {
  font-size: 16px;
  color: var(--color-secondary);
  margin: 0;
}

<p
  className="emotion-0"
/>
`;

exports[`Text TextSm matches snapshot 2`] = `
.emotion-0 {
  font-size: 12px;
  color: var(--color-secondary);
}

<small
  className="emotion-0"
/>
`;
